examples/006_filter_composite_data_example.py

```diff
@@ -99,16 +101,37 @@
 # Using the :func:`.get_ply_wise_data` function has the advantage that all the averaging
 # and filtering is done on the server side.
 if version_equal_or_later(server, "8.0"):
-    elemental_max = get_ply_wise_data(
-        field=stress_field,
+    elemental_values = get_ply_wise_data(
+        field=stress_field_material_coord,
         ply_name="P1L1__ud_patch ns1",
         mesh=composite_model.get_mesh(),
         component=Sym3x3TensorComponent.TENSOR11,
         spot_reduction_strategy=SpotReductionStrategy.MAX,
         requested_location=dpf.locations.elemental,
     )
 
-    composite_model.get_mesh().plot(elemental_max)
+    composite_model.get_mesh().plot(elemental_values)
+
+# %%
+# The results can also be requested in global coordinates. This example gets the stress values in
+# the global coordinate system, selects the top spot of a selected ply and averages the values
```

Comment:
```suggestion
# the global coordinate system, selects the top spot of a selected ply, and averages the values
```
We use Oxford commas, which is merely a comma before the "and" or the "or" in a list of three or more items.